### ISSUE TITLE
Update the performance comparison section of the IO docs

### ISSUE THREAD

**Issue by jorisvandenbossche** [python-sprints/pandas-mentoring#163]
In the IO doc page of the user guide, there is a section comparing the performance of some the supported IO methods: https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#performance-considerations

This is already quite old, and would be nice to update this:

- add the parquet format to the compared set
- rerun the code and update the code-block with your runs

**Comment by WuraolaOyewusi** [comment]
@jorisvandenbossche  @datapythonista 
Hello, Is this still open?
I've checked pandas but i'm just asking again.
If it is. I will like to do this

**Comment by datapythonista** [comment]
Yes, it'll be great if you can work on this. You can see the latest version of the doc here: https://dev.pandas.io/docs/user_guide/io.html#performance-considerations

Let me know if you need any help, and you can tag me when you open the PR, if you want a review.

Thanks @WuraolaOyewusi!

**Comment by WuraolaOyewusi** [comment]
Ok marc. I will do just that.
